AndroidSDKCore/src/main/java/com/leanplum/Leanplum.java

```diff
@@ -1924,6 +1941,9 @@ public static void forceContentUpdate(final VariablesChangedCallback callback) {
       Map<String, Object> params = new HashMap<>();
       params.put(Constants.Params.INCLUDE_DEFAULTS, Boolean.toString(false));
       params.put(Constants.Params.INBOX_MESSAGES, LeanplumInbox.getInstance().messagesIds());
+      if (LeanplumInternal.getIsVariantDebugInfoEnabled()) {
```

Comment:
You can rewrite this as
params.put(Constants.Params.INCLUDE_VARIANT_DEBUG_INFO, LeanplumInternal.getIsVariantDebugInfoEnabled());